.github/workflows/release_to_pypi.yml

```diff
@@ -41,6 +41,16 @@ jobs:
           draft: false
           prerelease: false
 
+      - name: Create Release Notes
```

Comment:
I suppose that this will conflict with Create GitHub Release above

Reply:
> I suppose that this will conflict with Create GitHub Release above

Yeah, more than likely. We will have to work that out iteratively seems like.

Reply:
We need one or the other. Also note that removing the one above will break the last one which uses `steps.create_release.outputs.upload_url`. In any case, `actions/create-release` is archived so we shall find a new way of doing things.

Reply:
I actually think things are more simple than I thought. We can simply prune down the current `momepy` action to get our desired result. That final step []()

We can see in the [`v0.7.0` release](https://github.com/pysal/momepy/releases/tag/v0.7.0) there are 3 assets there. 
* `momepy-0.7.0.tar.gz`
* `Source code (zip)`
* `Source code (tar.gz)`

However, `momepy-0.7.0.tar.gz` and `Source code (tar.gz)` are the exact same thing, just different names - that's what [*Upload Release Asset (sdist) to GitHub*](https://github.com/pysal/momepy/blob/e6b0abf4c195d5c7908355e624cce96fd83bde02/.github/workflows/release_to_pypi.yml#L50) is doing (and it's not needed).

I'm going to make some adjustments and push up (what I hope is) the fix.